src/wallet/rpcwallet.cpp

```diff
@@ -2537,13 +2537,13 @@ UniValue getlabeladdress(const UniValue& params, bool fHelp)
 /** Convert CAddressBookData to JSON record.
  * The verbosity of the output is configurable based on the command.
  */
-static UniValue AddressBookDataToJSON(const CAddressBookData& data, bool includeName, bool includeDestData)
+static UniValue AddressBookDataToJSON(const CAddressBookData& data, bool verbose)
 {
     UniValue ret(UniValue::VOBJ);
-    if (includeName)
+    if (verbose)
```

Comment:
Does the order of push_back matter here? Might be cleaner to lump all `if (verbose)` under one branch...